ecosystem/typescript/sdk_v2/src/utils/const.ts

```diff
@@ -0,0 +1,3 @@
+import { Network } from "./api-endpoints";
+
+export const DEFAULT_NETWORK = Network.TESTNET;
```

Comment:
I'd default this to either `DEVNET` or `MAINNET`, but probably let's do `DEVNET` until we launch it or make another decision.

Reply:
done